docs/database/database-spans.md

```diff
@@ -71,6 +71,8 @@ The <span id="target-placeholder">`{target}`</span> SHOULD adhere to one of the
 - `server.address:server.port`
 - `db.system`
 
+A given instrumentation SHOULD strive for consistency in span names. This means that if a specific attribute from the [`{target}`](#target-placeholder) list is selected, it should always be used for the span name. If certain queries lack this specific attribute, the instrumentation SHOULD use `{db.operation.name}` as the span name and omit `{target}`, rather than falling back to the next attribute in the [`{target}`](#target-placeholder) list.
```

Comment:
can you please break the lines? That way it's easier to leave comments and suggestions when the lines are between 80-120 chars long. 

We have tooling to enforce this but not running it at the moment.